src/js/components/Page.js

```diff
@@ -179,6 +180,23 @@ var Page = React.createClass({
     );
   },
 
+  getLicensingBanner() {
+    // todo use mountservice and mount LicenseBanner
+    // which will render the Banner as a child
+    // return <MountService.Mount type="Page:LicenseBanner" />
```

Comment:
In the respective EE PR it has `DCOS:*` prefix. I think we should come up with a naming scheme for Mounts.

Reply:
@nLight I'd prefer to use shared types instead of introducing a naming scheme. Anyway, could you please file a foundation issues to revisit the mount service?

Reply:
It's there already ;) https://jira.mesosphere.com/browse/DCOS-18779 one of the in the epic https://jira.mesosphere.com/browse/DCOS-18778